tools/cli/help.txt

```diff
@@ -941,6 +941,12 @@ Pass --create to create a new package.
 Options:
   --create   publish a new package
   --update   changed metadata of a previously published version
+  --release <version>
+             resolve top-level version conflicts by specifying the release
+             version. This option is particularly useful during the migration
+             process when dealing with packages that have conflicting version
+             requirements. Specify the version that satisfies all package
+             version constraints.
```

Comment:
This description is correct, but it describes one of the few side effects of using the `--release` option. The description should probably start with describing what the option does: `--release        Specify the release of Meteor to use` (taken from the `meteor run` help).